src/main/resources/admin/tools/content-type-manager/modules/module-example.html

```diff
@@ -17,6 +17,18 @@
       return recursivelyFindPathOfFormElement(jsonAtPath, nextPath);
     };
 
+    const renameAttributeNameIfNeeded = (jsonAsString, schemaJson) => {
+      const schemaAttributeName = (schemaJson.attributes || {}).name || "";
+      if (
+        schemaAttributeName &&
+        jsonAsString.indexOf(schemaAttributeName) !== -1
+      ) {
+        const counter =
+          jsonAsString.split(`"name":"${schemaAttributeName}`).length - 1;
```

Comment:
String spliting stringyfied json is not a good idea... be careful with this 
`"name:": "schemaName"` is valid 
`"name:":"schemaName"` is also valid

Reply:
Makes sense! Any suggestions to change the way I'm counting the occurrences?

Reply:
I don't understand the traversal of the jsonstring / json object and how you are using string paths to access the json elements. So i cant say i understand the spliting of the jsonstring with the amount of names -1. 
Would have to spend some time coding it myself and i don't think i have time for that.